beam-core/beam-core.cabal

```diff
@@ -56,29 +56,32 @@ library
                        Database.Beam.Query.Relationships
 
                        Database.Beam.Schema.Lenses
-  build-depends:       base         >=4.7     && <5.0,
+  build-depends:       base         >=4.9     && <5.0,
```

Comment:
Are the old lower bounds wrong or something? Ideally we wouldn't have to raise them just to support a barely released GHC.

Reply:
`beam-core` requires GHC-8.0, which comes with `base-4.9`.

Other bounds are unsatisfiable as well.

Reply:
Does it not compile with the lower bound? That should be a bug filed against cabal or stack.

Reply:
@tathougies I don't understand your comment? `>=4.7` is "wrong" as it gives impression that `beam-core` should be buildable with GHC-7.8 (base-4.7), which isn't the case.

Reply:
So beam no longer would compile on 7.8, due to the changes in base, is that right? That's fine with me of course

Reply:
You started using `TypeApplications` which is GHC-8.0+ language extension.

Reply:
Good point!